models/op/optimism/op_optimism_airdrop_1_claims.sql

```diff
@@ -0,0 +1,30 @@
+{{
+    config(
+        alias='optimism_1',
+        materialized = 'incremental',
+        file_format = 'delta',
+        incremental_strategy = 'merge',
+        unique_key = ['recipient', 'tx_hash', 'evt_index'],
+        post_hook='{{ expose_spells(\'["optimsm"]\',
+                                "project",
+                                "airdrop",
+                                \'["hildobby"]\') }}'
+    )
+}}
+
+SELECT 'optimism' AS blockchain
+, evt_block_time AS block_time
+, evt_block_number AS block_number
+, 'Optimism' AS project
+, 'Optimism Airdrop 1' AS airdrop_identifier
+, account AS recipient
+, contract_address
+, evt_tx_hash AS tx_hash
+, amount/POWER(10, 18) AS quantity
```

Comment:
do we need to cast any of these as double for dunesql future-proofing? Many had decimal amounts (i.e. 778.79)